test/unit/lib/plugins/aws/invoke.test.js

```diff
@@ -573,18 +500,28 @@ describe('test/unit/lib/plugins/aws/invoke.test.js', () => {
     expect(lambdaInvokeStub).to.have.been.callCount(0);
   });
 
-  xit('TODO: should fail the process for failed invocations', async () => {
+  it('should fail the process for failed invocations', async () => {
+    const lambdaInvokeStub = sinon.stub();
     await expect(
       runServerless({
         fixture: 'invocation',
         command: 'invoke',
         options: {
           function: 'callback',
-          path: 'not-existing.yaml',
+        },
+        awsRequestStubMap: {
+          Lambda: {
+            invoke: (args) => {
+              lambdaInvokeStub.returns({
+                Payload: args.Payload,
+                LogResult: Buffer.from('test').toString('base64'),
+                FunctionError: true,
+              });
+              return lambdaInvokeStub(args);
+            },
+          },
         },
       })
-    ).to.eventually.be.rejected.and.have.property('code', 'TODO');
-    // Replace
-    // https://github.com/serverless/serverless/blob/537fcac7597f0c6efbae7a5fc984270a78a2a53a/test/unit/lib/plugins/aws/invoke.test.js#L317-L332
+    ).to.be.eventually.rejectedWith(Error, 'Invoked function failed');
```

Comment:
it looks like the ideal approach is to check the code property but I did it this way to replicate the test that it's replacing and to avoid duplicating a previous `file not found ` test case.

Reply:
I think it's fine for now as in this case the `Error` being thrown doesn't have the code property, but it's outside the scope of this PR